packages/tooling/fast-figma-plugin-msft/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Fluent UI Design Tokens (Alpha)",
+  "name": "Fluent UI WC Designer (Alpha)",
   "api": "1.0.0",
   "main": "./dist/main.js",
   "ui": "./dist/index.html",
-  "id": "796902094715412666",
+  "id": "1081287243644186292",
```

Comment:
I did learn we can get Figma to manually transfer plugin ownership if that is preferred. It's up to you.

Reply:
Good to know. Since we don't need to migrate data right now, I think we don't need to do that this time.